public_html/wp-content/themes/pattern-directory/src/components/active-filter-bar/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+import { store } from '@wordpress/viewport';
+import { Flex, FlexItem } from '@wordpress/components';
+
+function ActiveFilterBar( { children, actionsTitle, actions } ) {
+	const isSmall = select( store ).isViewportMatch( '< medium' );
```

Comment:
_This is a minor thing, feel free to disregard it if it doesn't feel good to you_.

```suggestion
	const isGreaterThanSmall = select( store ).isViewportMatch( '>= small' );
```

This would allow the condition below to be `{ isGreaterThanSmall...` instead of `{ ! isSmall ...`. It feels more readable to me without the negative. 

> We may want to use a more composite approach to the component; breaking out the "actions".

That sounds good to me, and would allow using `ifViewportMatches` here.

Reply:
Okay, i'll redo.